src/components/SnapshotDetailView.jsx

```diff
@@ -100,7 +100,7 @@ export class SnapshotDetailView extends React.PureComponent {
     const { snapshot } = this.props;
     const datasets = snapshot.source.map((s) => s.dataset);
     const filtered = datasets.filter((d) =>
-      d.name.toLowerCase().includes(searchString.toLowerCase()),
+      d.name.toLowerCase().includes(searchString ? searchString.toLowerCase() : ''),
```

Comment:
thanks!  I agree right now it can't but if ever the value gets programmatically set to null (e.g. if we move to back it with a store and inadvertently set it it null).  I just like the safety of it.
I typically do something like:
```suggestion
      d.name.toLowerCase().includes((searchString || '').toLowerCase(),
```

or I think you could even do something like:
```suggestion
      d.name.toLowerCase().includes(searchString?.toLowerCase(),
```

Reply:
Oh right - I think the second (null coalescing)  wouldn't work in our case b/c if searchString is undefined, then it would return undefined and the .includes() call would probably  be unhappy w/ that input. 
The `or` operator should work.